src/link/main.c

```diff
@@ -108,6 +109,8 @@ static struct option const longopts[] = {
 	{ "version",      no_argument,       NULL, 'V' },
 	{ "verbose",      no_argument,       NULL, 'v' },
 	{ "wramx",        no_argument,       NULL, 'w' },
+	/* x is reserved for nopad but is not a short argument */
```

Comment:
If you're reserving `x` for no padding, why not just let it be a short argument?

Reply:
I figured that `x` wouldn't be a very descriptive letter. All the others at least have some relation to their respective option but this one has none.

Reply:
There's still no benefit in forbidding the short form if you're going to reserve the letter anyway.

Reply:
If I'm reading the code correctly, the short option isn't even forbidden, but merely undocumented.

Note that you can use unprintable characters for this.

Reply:
@NieDzejkob It's not in the optstring, so it won't be recognized. Anyway, I have a commit ready to fix this.

Reply:
done, in case you haven't noticed.

Reply:
It should've been automatically folded by GitHub, since the code is now marked as outdated.